server/controllers/visitation.controllers.js

```diff
@@ -55,12 +55,13 @@ const VisitationController = {
   rescheduleVisitation(req, res, next) {
     const visitationInfo = req.locals;
     const { user } = req;
-    rescheduleVisitation({ user, visitationInfo })
+    const action = 'reschedule';
+    processVisitation({ user, visitationInfo, action })
       .then(({ visitation, mailDetails, property }) => {
-        const contentTop = `The visitation to ${property.name} on ${
-          visitation.rescheduleLog[0].rescheduleFrom
-        }, has been rescheduled for ${visitation.visitDate} by ${
-          user.vendor.companyName ? user.vendor.companyName : `${user.lastName} ${user.firstName}`
+        const contentTop = `The visitation to ${property.name} on ${convertDateToWords(
+          visitation.rescheduleLog[0].rescheduleFrom,
+        )}, has been rescheduled for ${convertDateToWords(visitation.visitDate)} by ${
+          user.vendor.companyName ? user.vendor.companyName : `${user.firstName} ${user.lastName}`
```

Comment:
Lets create a function for this so that it would be easier to update the name format in the future. 

```suggestion
          user.vendor.companyName ? user.vendor.companyName : `${getUserName(user)}`
```